doc/release-notes-19093.md

```diff
@@ -0,0 +1,7 @@
+RPC changes
```

Comment:
Should the filename have the PR number? (19339) I didn't know about this convention yet, just looked it up: https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#release-notes